src/main/java/seedu/address/model/person/UniquePersonList.java

```diff
@@ -97,6 +97,13 @@ public void setPersons(List<Person> persons) {
         internalList.setAll(persons);
     }
 
+    /**
+     * Deletes meetings with times that are before the local time on machine.
+     */
+    public void syncMeetingTimes() {
+        internalList.forEach(Person::syncMeetingTimes);
+    }
+
```

Comment:
Since this is done on the UniquePersonList, SyncCommand should work on the original list and not just the currently displayed list right?
Correct me if I'm wrong

Reply:
yep. it syncs the entire FABook